Sources/MapboxDirections/Congestion.swift

```diff
@@ -37,3 +37,8 @@ public enum CongestionLevel: String, Codable, CaseIterable {
      */
     case severe
 }
+
+/**
+ `NumericCongestionLevel` is the level of traffic congestion along a road segment in numeric form, from 0-100. A value of 0 indicates no congestion, a value of 100 indicates maximum congestion. Entries may be `nil` if congestion on that segment is not known.
+ */
+public typealias NumericCongestionLevel = Int
```

Comment:
Maybe it should be `Int?` as you say:
> Entries may be `nil` if congestion on that segment is not known.

Reply:
Making a type alias of an optional makes the type a bit less clear to use. Concerning doc comment - good catch, I've moved it to `segmentNumericCongestionLevels` doc.